index.css

```diff
@@ -78,8 +88,372 @@
+  }
+
+  .project-list .second-list {
+    margin-left: -20%;
+  }
+
+  .project-list .third-list {
+    margin-left: -8%;
+  }
+
+  .works-centered button {
+    display: flex;
+    align-items: left;
+    font-size: 200%;
+    margin-left: 0%;
+    margin-right: 60%;
+    padding: 30px;
+    cursor: pointer;
+    border: none;
+    background-color: #ff6b00;
+    font-weight: bold;
+    color: white;
+  }
+
+  .print-data button {
+    border: none;
+    background-color: #ff6b00;
+    cursor: pointer;
+    font-weight: bold;
+    color: white;
+    font-size: 200%;
+  }
+
+  .interesting button {
+    margin-left: 30%;
+    margin-right: 30%;
+    border: none;
+    cursor: pointer;
+    background-color: #ff6b00;
+    font-weight: bold;
+    color: white;
+    font-size: 200%;
+    padding: 4% 4% 4% 4%;
+    margin-bottom: 22%;
+  }
+
+  .about-me button {
+    border: none;
+    background-color: #ff6b00;
+    cursor: pointer;
+    font-weight: bold;
+    color: white;
+    font-size: 200%;
+    margin-left: 8%;
+    margin-right: 50%;
+    padding: 4% 4% 4% 4%;
+    margin-bottom: 70%;
+  }
+
+  .works-centered button:hover {
+    background-color: white;
+    color: black;
+    border: solid;
+  }
+
+  .print-data {
+    display: grid;
+    height: 100vh;
+    background-image: url("./logos/Professional Printing Data.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    margin-top: 5%;
+  }
+
+  .print-data h3 {
+    font-family: 'Crete Round', serif;
+    color: white;
+    font-size: 400%;
+    text-align: left;
+    margin-top: 24%;
+    margin-left: 5%;
+  }
+
+  .print-data p {
+    font-family: 'Inter', sans-serif;
+    color: white;
+    font-size: 200%;
+    margin-top: -15%;
+    margin-left: 5%;
+  }
+
+  .print-data .list-x {
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    list-style: none;
+    margin-bottom: 10px;
+    margin-top: -25px;
+  }
+
+  .list-x .start-list-x {
+    margin-left: -7%;
+  }
+
+  .list-x .second-list-x {
+    margin-left: 15%;
+  }
+
+  .list-x .third-list-x {
+    margin-left: 20%;
+  }
+
+  .print-data button:hover {
+    background-color: white;
+    color: black;
+    border: solid;
+  }
+
+  .about-me {
+    background-color: rgba(28, 26, 25, 1);
+    margin-top: 33%;
+  }
+
+  .background-holder {
+    display: flex;
+    flex-direction: column;
+    background-image: url("./logos/About Me.png");
+    background-repeat: no-repeat;
+    background-size: 300px 500px;
+    background-position: right bottom;
+  }
+
+  .about-me h2 {
+    color: rgba(255, 107, 0, 1);
+    font-size: 800%;
+    font-family: 'Crete Round', serif;
+    text-align: center;
+    padding-top: 35%;
+    margin-left: 8%;
+    margin-right: 8%;
+  }
+
+  .about-me p {
+    font-family: 'Inter', sans-serif;
+    font-size: 200%;
+    margin-left: 8%;
+    margin-right: 8%;
+    margin-top: -10%;
+    margin-bottom: 8%;
+  }
```

Comment:
- [ ] Kindly use pixels instead percentages for the font size and margins to be similar to the Figma design.